loops_lab.js

```diff
@@ -0,0 +1,57 @@
+
+
+
+//-------------------------------------
+let num = 5;
+while (num < 25) {
+  console.log(num);
+  num += 1;
+}
+
+for (let number = 0; number <= 10; number += 1) {
+  if (number % 2 !== 0) {
+    console.log(number + ' num is odd');
+  } else {
+    console.log(number + ' num is even');
+  }
+}
+
+let bynine = 99;
+while (bynine > 0) {
+  if (bynine % 9 !== 0) {
+    console.log(bynine  + ' is not divisible by 9');
+  } else {
+    console.log(bynine + 'is divisible by 9');
+  } bynine -= 1;
+}
+
+for (let i = 5; i <= 100; i += 10) {
+  console.log(i);
+}
+
+//5.It would run infinite
+
+let str1 = '';
```

Comment:
Code should like this: 
1
2
Fizz
4
Buzz
........
14
FizzBuzz
16

Your code will give something completely different. Next time, please read the question carefully and run the code before submitting it.